docs/src/reference/implementations-tinkergraph.asciidoc

```diff
@@ -27,7 +27,7 @@ limitations under the License.
 ----
 
 image:tinkerpop-character.png[width=100,float=left] TinkerGraph is a single machine, in-memory (with optional
-persistence), non-transactional graph engine that provides both OLTP and OLAP functionality. It is deployed with
+persistence), transactional graph engine that provides both OLTP and OLAP functionality. It is deployed with
```

Comment:
Not sure about this change as base TinkerGraph is still non-transactional. I think the`Transactions` subsection is good enough.

Reply:
Yea I wasn't sure how this would interact here. Maybe I'll revert